test_conformance/extensions/cl_mutable_dispatch_config_khr/main.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2022 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "procs.h"
+#include "harness/testHarness.h"
+
+test_definition test_list[] = {
+    ADD_TEST(mutable_dispatch_device_query),
+    ADD_TEST(mutable_dispatch_command_buffer),
```

Comment:
I think these should start `mutable_command_info` rather than `mutable_dispatch`, we don't need the mutable_dispatch prefix since that's already in the test executable name, but knowing these test `clGetMutableCommandInfoKHR` is helpful